src/components/Question/Question.tsx

```diff
@@ -0,0 +1,157 @@
+    question,
+    updateFundingApplicationQuestion
+}) => {
+    const initialState = {
+        complete: false,
+        applicationCase: ""
+    };
+
+    const q = question.getFundingApplicationQuestion;
+
+    const [questionForm, setQuestionForm] = useState(initialState);
+
+    const [validForm, setValidForm] = useState<boolean>(true);
+
+    const wordLimit = (q && q.wordLimit) || 0;
+
+    const [wordsRemaining, setWordsRemaining] = useState(wordLimit);
+
+    const [formState, { textarea, checkbox }] = useFormState(initialState);
+
+    const onSubmit = useCallback(
+        async (event: FormEvent) => {
+            event.preventDefault();
+
+            updateFundingApplicationQuestion(
+                formState.values.applicationCase,
+                formState.values.complete
+            );
+        },
+        [
+            formState.values.applicationCase,
+            formState.values.complete,
+            updateFundingApplicationQuestion
+        ]
+    );
+
+    function onInputChange(event: string) {
+        const textAreaValue = event;
+        if (textAreaValue.length <= wordLimit) {
+            const newWordsRemaining = wordLimit - event.length;
+            setWordsRemaining(newWordsRemaining);
+            setQuestionForm({
+                ...questionForm,
+                applicationCase: textAreaValue
+            });
+            setValidForm(true);
+        } else {
+            const newWordsRemaining = wordLimit - event.length;
+            setWordsRemaining(newWordsRemaining);
+            setValidForm(false);
+            return "Invalid";
+        }
+    }
+
+    return (
+        <>
+            <Breadcrumbs>
+                <Breadcrumbs.Link href={`/`}>Back</Breadcrumbs.Link>
+            </Breadcrumbs>
+
+            <LoadingBox loading={false}>
+                <Caption mb={1}>ji </Caption>
+                <Title mb={7}>Case for support</Title>
+
+                <Caption size="M" mb={1}>
+                    Provide your case for support
+                </Caption>
+                <Details
+                    mb={3}
+                    summary="What should I include in case for support section?"
+                >
+                    ...
+                </Details>
+                <form onSubmit={onSubmit}>
+                    <FormGroup error={!validForm}>
+                        {!validForm && (
+                            <ErrorText>
+                                {" "}
+                                Please ensure the text is within the word limit{" "}
+                            </ErrorText>
+                        )}
+                        <TextArea
+                            {...textarea({
+                                name: "applicationCase",
+                                validate: onInputChange
+                            })}
+                        />
+                    </FormGroup>
+
+                    <div>Words remaining: {wordsRemaining}</div>
```

Comment:
What's the div for? Use a govuk component for all text, or it doesn't get fonts applied.